src-java/kilda-model/src/main/java/org/openkilda/model/Flow.java

```diff
@@ -395,6 +395,17 @@ public boolean isLooped() {
         return getLoopSwitchId() != null;
     }
 
+    /**
+     * Returns Y point switch id.
+     */
+    public SwitchId getYPointSwitchId() {
+        YFlow yFlow = getYFlow();
+        if (yFlow == null) {
+            return null;
+        }
+        return yFlow.getYPoint();
+    }
```

Comment:
[minor], here you can do something like: 

return yFlow == null ? null : yFlow.getYPoint();